samtranslator/validator/validator.py

```diff
@@ -1,55 +1,259 @@
 import json
+import re
 
 import jsonschema
 from jsonschema.exceptions import ValidationError
 
 from . import sam_schema
 
 
-class SamTemplateValidator(object):
+class SamTemplateValidator:
```

Comment:
Can we add type hinting to this file?

Reply:
I don't think we use type hinting in the translator project because of Python2.